sources/web/datalab/static/datalab.js

```diff
@@ -240,7 +240,7 @@ function initializePage(dialog, saveFn) {
         path = path.substr('/notebooks'.length);
         path = path.substr(0, path.lastIndexOf('/'));
       }
-      path = encodeURIComponent('/content' + path);
+      path = '/content' + path;
```

Comment:
That makes sense, but can you send it as a separate PR?

Otherwise, the reason behind this change will get lost.